### PR TITLE
Add Atulpriya Sharma as co-chair of WG Platforms

### PR DESCRIPTION
**Pull request by krumware**
This officially adds Atulpriya Sharma as a co-chair of the Platforms WG.
Welcome @techmaharaj! 🎉🎉

(also, this updates some of the localization content in the readmes)

### CODE REVIEW

File: platforms-wg/README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ To register for updates and to "Join" the group, please also consider joining th
 
 ## Chairs
 
-* Josh Gavant (@joshgav)
-* Roberth Strand (@roberthstrand)
+* Technical Lead - Roberth Strand (@roberthstrand)
```
Comment (abangser) [inline platforms-wg/README.md:12]: IMO this should be a different section as TL and chair are actively different. 

(And of course for all languages 🙈 though this language seems to be missing the meeting updates)

File: website/content/ja/wgs/platforms/_index.md [hunk 2]
```diff
@@ -9,19 +9,21 @@ To participate join us on Slack at
...
+* Atulpriya Sharma (@techmaharaj)
 
 ## Meetings
 
 * Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-    * [2nd Tuesday event](https://www.google.com/calendar/event?eid=cWE0ZHMyMThicDF1dDZsa3N2ZGlqYmt2Z2pfMjAyNDA1MTRUMTUwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&ctz=UTC)
-    * [4th Tuesday event](https://www.google.com/calendar/event?eid=NGhyOHY1ZWVrbDliODY3bXU5ZnRtYWo0ZGdfMjAyNDA1MjhUMTYwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&ctz=UTC)
-    * [Full CNCF calendar](https://calendar.google.com/calendar/u/0/embed?src=linuxfoundation.org_o5avjlvt2cae9bq7a95emc4740@group.calendar.google.com)
+* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
+* Meeting Location, Zoom: https://zoom-lfx.platform.linuxfoundation.org/meeting/94326372364?password=0681764d-b152-4295-8bd2-6b9fcf77bff1
```
Comment (abangser) [inline website/content/ja/wgs/platforms/_index.md:25]: This seems to duplicate the row below?

File: website/content/ja/wgs/platforms/_index.md [hunk 3]
```diff
@@ -9,19 +9,21 @@ To participate join us on Slack at
...
 * Zoom: https://zoom-lfx.platform.linuxfoundation.org/meeting/94326372364?password=0681764d-b152-4295-8bd2-6b9fcf77bff1
     * Passcode: 77777
 * Agendas and notes: <https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg>
+* Previous meetings can be viewed on the [TAG YouTube Channel](https://www.youtube.com/watch?v=eZYSQnsWRco&list=PLjNzvzqUSpxKH8X7wNfYZtkH_ARSeeQH0)
```
Comment (abangser) [inline website/content/ja/wgs/platforms/_index.md:29]: Very nice to add this! Just missing on a couple of languages I think 🤔

**Review by abangser — CHANGES_REQUESTED** [review]
This is looking right, just not evenly applied across languages. When it's the same across all we should be good!

File: website/content/ja/wgs/platforms/_index.md (hunk 3 above)
Comment (krumware) [reply website/content/ja/wgs/platforms/_index.md:29]: it's on the other languages, I just think it wasn't there on all of them, so the other languages might not show the diff

**Review by roberthstrand — APPROVED** [review]

**Comment by techmaharaj** [comment]
Thanks @krumware and the entire team for this. Looking forward to contributing and helping the group achieve greater heights.

**Review by lianmakesthings — APPROVED** [review]
LGTM. @abangser can you approve your review as well so we can merge?

Just saying the branch naming (`chore/atul`) did Atul dirty 🤣

**Review by abangser — APPROVED** [review]

**Comment by abangser** [comment]
Sorry, on holiday, the delay isn't a reflection of my excitement for Atul to join us!!

**Pull request merged**